src/main/java/teammates/common/util/ActivityLogEntry.java

```diff
@@ -506,12 +503,13 @@ public static String generateServletActionFailureLogMessage(HttpServletRequest r
         String courseId = HttpRequestHelper.getValueFromRequestParameterMap(req, Const.ParamsNames.COURSE_ID);
         String studentEmail = HttpRequestHelper.getValueFromRequestParameterMap(req, Const.ParamsNames.STUDENT_EMAIL);
         ActivityLogEntry exceptionLog = new ActivityLogEntry(action, Const.ACTION_RESULT_FAILURE, null, message,
-                                                             url, courseId, studentEmail);
+                                                             url, courseId, studentEmail, userType);
         
         return exceptionLog.generateLogMessage();
     }
 
-    public static String generateSystemErrorReportLogMessage(HttpServletRequest req, EmailWrapper errorEmail) {
+    public static String generateSystemErrorReportLogMessage(HttpServletRequest req, EmailWrapper errorEmail,
+                                                            UserType userType) {
```

Comment:
Minor issue, but move this one space to the right so that `[U]serType` and `[H]ttpServletRequest` aligns.